### ISSUE TITLE
Problem with the node name.

### ISSUE THREAD

**Issue by Xianjin111** [lisiyao21/Bailando#41]
Thank you for your perfect work. I try to use the Choreographic for music in the wild. But when I run the command, I didn't know how to set the node name. Can anyone else help me?

Thank you very much.

**Comment by as501226107** [comment]
if fix it out?

**Comment by Jxxxnkai** [comment]
I have come across the same problem. Have u fixed this problem? If u know how to get the node name, can u tell me the method. Thank u

**Comment by DonlynLee** [comment]
Me too, have you found the solution?

**Comment by Jxxxnkai** [comment]
Actually, I didn't use this command. I have used this command 'python main.py --train --config configs/sep_vqvae_root.yaml'. You can try it. If u have other questions, I'm pleased to offer my idea.


江晓楠
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "lisiyao21/Bailando"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年1月20日(星期六) 下午4:01
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [lisiyao21/Bailando] Problem with the node name. (Issue #41)





 
Me too, have you found the solution?

**Comment by DonlynLee** [comment]
Sorry for bothering you, and thank you so much for your reply! I have tried the 'python xx.py' command instead. By the way, have you encountered the problem 'json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)' when running the code? I downloaded the preprocessed feature the author offered, and I check the json file is right.

**Comment by Jxxxnkai** [comment]
Yes. U can try to delete&nbsp;data/aistpp_train_wav/gPO_sBM_cAll_d10_mPO0_ch06.json. This is file is 0 KB. I have finished the step one



江晓楠
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "lisiyao21/Bailando"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年1月20日(星期六) 晚上10:48
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [lisiyao21/Bailando] Problem with the node name. (Issue #41)





 
Sorry for bothering you, and thank you so much for your reply! I have tried the 'python xx.py' command instead. By the way, have you encountered the problem 'json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)' when running the code? I downloaded the preprocessed feature the author offered, and I check the json file is right.

**Comment by DonlynLee** [comment]
Thank you so much 555! I will try it again later. If you have any problem, we can communicate together. Wish you all the best~

**Comment by Jxxxnkai** [comment]
Sorry for bothering you. Have u finished the first step(VQ-VQE without global velocity)?&nbsp; I completed the first step. But when I train&nbsp; glabal velocity branch of pose VQ-VAE, I come across the result 'Killed'. Do u have this problem?




江晓楠
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "lisiyao21/Bailando"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年1月20日(星期六) 晚上10:48
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [lisiyao21/Bailando] Problem with the node name. (Issue #41)





 
Sorry for bothering you, and thank you so much for your reply! I have tried the 'python xx.py' command instead. By the way, have you encountered the problem 'json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)' when running the code? I downloaded the preprocessed feature the author offered, and I check the json file is right.

**Comment by DonlynLee** [comment]
Hi, actually I didn't run the training stage, I downloaded the pretrained models to test first. I haven't met the problem, but when I test the VQ-VAE(first step of test), I encountered the problem that the code is stuck at the "Generating Images"...

**Comment by XUCHUANXINA** [comment]
Hi,I want to ask about the paper "Bailando". I used the preprocessed data provided by the author and the project code on GitHub to reproduce the paper. All hyperparameters were set as per the original author's configuration. I conducted the reproduction on PyTorch 2.0.0, Python 3.8, CUDA 11.8, and an RTX 3090. However, the reproduction results differ significantly from those of the author. What could be the reasons for this? Could you offer some assistance? Thank you.

**Comment by Jxxxnkai** [comment]
Actually, my team has finished reproduction of this paper in 2023. It is  2 years ago, so I nearly forgot some details of this paper. However, I remembered that my teams' results were not good enough. We guessed that the reason of that was about datasets, because we cut down some data in this datasets to get the result as quick as enough to integrate a project with this paper. By the way, our GPU is Tesla V100. 
About other reasons, it may be related to some parameters in the training process. I think you can find more reasons with the assistance of ai tools, like cursor hhhhh. We didn't have this tools at that time. I believe that these tools will help u.

**Comment by XUCHUANXINA** [comment]
Okay, thank you very much for your answer, this problem has bothered me for a long time, I also tried to apply TSL V100 for training, but the effect is still not as good as the evaluation indicators given by the author, I will continue to investigate, thank you for your reply.
